src/pages/management/configuration-api/v2.0/index.mdx

```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
 
 <Text>
 
+### Ads traffic
+
+The greeting will be sent when the customer first visit url contains specified ads parameters.
+
+When using the `ads_traffic` greeting rule, you also may provide `variable_name` and `variable_value`.
+For example, providing `variable_name` as `facebook` and `variable_value` as `1234` will send the greeting when
+the customer first visit url contains `fbclid=1234`.
+You also can add the optional `operator` parameter (defaults to `equals`).
+
+Possible `variable_name` values are:
+<ul>
+  <li><code>facebook</code> for <code>fbclid</code>,</li>
+  <li><code>google</code> for <code>dclid</code>, <code>gbraid</code>, <code>gclid</code>, <code>gclsrc</code>, <code>
+  wbraid</code>,</li>
+  <li><code>microsoft</code> for <code>msclkid</code>,</li>
+  <li><code>twitter</code> for <code>twclid</code>,</li>
+  <li><code>yahoo</code> for <code>yclid</code>,</li>
+</ul>
+
+Max allowed `variable_value` length is 255 characters.
+
+When not providing `variable_name` and `variable_value`, the greeting will be sent when the customer first visit url
```

Comment:
Like above, I believe we want
When not providing `variable_name` and `variable_value`, the greeting will be sent when the customer first visits the URL containing any of the ads parameters.